XML/TestCases/FunctionalTests.xml

```diff
@@ -968,4 +968,17 @@
 	<SetupType>OneVM</SetupType>
 	</SetupConfig>
 </test>
+<test>
+  <testName>VERIFY-UIO-DRIVER</testName>
+  <testScript>verify_uio_driver.sh</testScript>
+  <files>.\Testscripts\Linux\verify_uio_driver.sh,.\Testscripts\Linux\utils.sh</files>
+  <Platform>Azure,HyperV</Platform>
+  <Category>Functional</Category>
+  <Area>UIO</Area>
```

Comment:
UIO -> CORE. We keep all vmbus, cpu tests in CORE area.

Reply:
Hi @juhlee-microsoft ,
Sorry currently we may not have enough knowledge base and energy to debug the case in ubuntu/SLES. But I'll add the distro condition in the cases to prevent failure in other distros. Thank you so much for noticing!